src/InverseCdf.cpp

```diff
@@ -0,0 +1,47 @@
+#include "InverseCdf.h"
+#include <math.h>
+#include "Rand.h"
+
+
+// set defaults if not present in parameter file
+void InverseCdf::set_defaults(double start_value)
+{
+	cdf_values_[CDF_RES] = start_value;
+	for (int i = 0; i < CDF_RES; i++)
+		cdf_values_[i] = -log(1 - ((double)i) / CDF_RES);
+}
+
+void InverseCdf::apply_exponent()
+{
+	for (int quantile = 0; quantile <= CDF_RES; quantile++)
+	{
+		cdf_values_[quantile] = exp(-cdf_values_[quantile]);
+	}
+}
+
+void InverseCdf::apply_exponent(double value)
+{
+	for (int i = 0; i <= CDF_RES; i++)
+	{
+		cdf_values_[i] = exp(value);
+	}
+}
+
```

Comment:
I'm not sure if there are better names for these three functions - perhaps the first one should include the neg_log somehow, and the third could use the word `assign` instead of `apply`

Reply:
Think this is a good shout. Happy to approve once this change has been made and once regression tests pass again.

Reply:
Done.